config_builder.go

```diff
@@ -1190,13 +1190,30 @@ func initNeutrinoBackend(cfg *Config, chainDir string,
 		return nil, nil, err
 	}
 
-	dbName := filepath.Join(dbPath, "neutrino.db")
-	db, err := walletdb.Create(
-		"bdb", dbName, !cfg.SyncFreelist, cfg.DB.Bolt.DBTimeout,
+	var (
+		db  walletdb.DB
+		err error
 	)
-	if err != nil {
-		return nil, nil, fmt.Errorf("unable to create neutrino "+
-			"database: %v", err)
+	switch {
+	case cfg.DB.Backend == kvdb.SqliteBackendName:
+		db, err = kvdb.Open(
+			kvdb.SqliteBackendName, context.Background(),
+			cfg.DB.Sqlite, dbPath, lncfg.SqliteNeutrinoDBName,
+			lncfg.NSNeutrinoDB,
+		)
+		if err != nil {
```

Comment:
Can move the err check outside of the `switch` and have the same additional context message for both.

Reply:
done